.github/workflows/build.yml

```diff
@@ -164,6 +169,32 @@ jobs:
               python3 script/build.py --build-type ${build_type} --build-type ${build_type}
               python3 script/archive.py --version ${build_version} --build-type ${build_type}
               echo "Produced artifact at ${PWD}/${artifact_name}"
+      - uses: uraimo/run-on-arch-action@v2.0.5
+        name: Test Build
```

Comment:
Do we need this?

The "Assemble" task looks as this task + check_release + archive

Reply:
Indeed, didn't noticed differences in `if:` conditions. It is good to have it.